railties/lib/rails/commands/test/test_command.rb

```diff
@@ -32,6 +32,31 @@ def perform(*)
         Rails::TestUnit::Runner.parse_options(args)
         Rails::TestUnit::Runner.run(args)
       end
+
+      # Define Thor tasks to avoid going through Rake and booting twice when using bin/rails test:*
+
+      Rails::TestUnit::Runner::TEST_FOLDERS.each do |name|
+        define_method(name) do |*|
```

Comment:
This never receives keyword arguments?

Reply:
No, it's basically an ARGV. It's called from Thor here: https://github.com/rails/thor/blob/v1.2.1/lib/thor/command.rb#L27